pmg/templates/layout.html

```diff
@@ -44,6 +44,7 @@
   {% assets "js" %}
     <script type="text/javascript" src="{{ ASSET_URL }}"></script>
   {% endassets %}
+  <script type="text/javascript" src="/static/resources/javascript/megamenu.js"></script>
```

Comment:
Please add this to assets.py, not here. That ensures it's included in the versioned asset bundle, is cached by the client, etc.